stestr/tests/test_config_file.py

```diff
@@ -49,7 +51,7 @@ def _check_get_run_command(self, mock_sys, mock_TestProcessorFixture,
         self.assertEqual(mock_TestProcessorFixture.return_value, fixture)
         mock_get_repo_open.assert_called_once_with('file',
                                                    None)
-        command = '%s -m stestr.subunit_runner.run discover -t "%s" "%s" ' \
+        command = '"%s" -m stestr.subunit_runner.run discover -t "%s" "%s" ' \
```

Comment:
It looks this change causes the error of `tox -e cover` because of `PYTHON=coverage run --source stestr` here.

https://travis-ci.org/mtreinish/stestr/jobs/581131380#L248
https://github.com/mtreinish/stestr/blob/master/tox.ini#L28

Reply:
Makes sense. So the callable may contain white spaces either because of the path or because of extra arguments. One reasonable workaround would be to check if the path exists, if so we can keep the quotes, otherwise we can assume that extra arguments are being used and the quotes should be dropped.